Sources/FoundationEssentials/Data/Data+Base64.swift

```diff
@@ -33,13 +43,10 @@ extension Data {
             return nil
         }
```

Comment:
Previously in #1160, you removed these `#if FOUNDATION_FRAMEWORK` path so we call into a unified implementation. Is it intentional that you aren't doing that here?